docs/examples/CustomBundle/Resources/config/form_extensions/brand/edit.yml

```diff
@@ -1,6 +1,6 @@
 extensions:
     pim-brand-edit-form:
-        module: pim/form/common/edit-form
+        module: custom_entity/form/common/edit-form
```

Comment:
For this line, I struggled with it during my training :P What is the benefit of creating our own module instead of using the pim one?

Reply:
To add [this listener](#diff-e360c59255f116572cf87cb69891a9b6R18)
Don't remember exactly what it does

Reply:
It allows to re-render the field after fetching data, that means that, if a field previously had a validation error message, and the form is re-submitted with valid data (at least for this field), the error message now disappears (it used to stay on the page before).